tests/e2e/tests/base/fix-one-issue.test.ts

```diff
@@ -5,6 +5,7 @@ import { generateRandomString } from '../../utilities/utils';
 import { KAIViews } from '../../enums/views.enum';
 import { FixTypes } from '../../enums/fix-types.enum';
 import * as VSCodeFactory from '../../utilities/vscode.factory';
+import { VSCodeDesktop } from '../../pages/vscode-desktop.page';
```

Comment:
This import can be removed, we shouldn't be using specific Vscode platforms in tests